languages/csharp/exercises/concept/randomness/.meta/design.md

```diff
@@ -0,0 +1,19 @@
+## Learning objectives
+
+- Know how to implement randomness with `System.Random` in C#.
+- Know that no seed is required
+- Know that both integers and real numbers can be generated
+- Know that once generated random numbers can be easily used for many purposes (including non-numeric ones).
+
+## Out of scope
+
+- Secure random numbers
+
+## Concepts
+
+- `randomness`: as they are handled in C# and .NET.
+- `basic-thread-safety`: ensure user is acquainted with the idea of thread safety. A deeper discussion can be deferred to the `concurrent-colletions` exercise.
```

Comment:
Why do we need to introduce thread safety here?

Reply:
We make the point that a good Random object is a static Random object.  If we are making that point then we should say that it is not thread-safe.  As we haven't introduced thread safety as a concept I  thought we should, particularly as we have to explain it.  I am more than happy to drop it from the concepts in this exercise and just discuss it in the _after.md_.

Reply:
> I am more than happy to drop it from the concepts in this exercise and just discuss it in the after.md.

I think I would prefer this. It could actually be in the learning objectives, as that can include the things in the `after.md` document too.

Reply:
Done